docker-compose.yml

```diff
@@ -19,6 +20,22 @@ services:
       - keyshade-dev
     volumes:
       - ./data/redis:/data
+
+  minio:
+    image: docker.io/bitnami/minio:2022
+    environment:
+      MINIO_ROOT_USER: minio
+      MINIO_ROOT_PASSWORD: supersecret
+    ports:
+      - '9000:9000'
+      - '9001:9001'
+    volumes:
+      - 'minio_data:/data'
```

Comment:
The data would be stored under `/data/minio`. Refer the other two containers to see how.

Reply:
noted